Header header
vision_bridge/ObjectInfo[] objects
